app/common.go

```diff
@@ -65,6 +26,28 @@ const (
 	ModelTypeObjectDetection
 )
 
+// ModelFramework is the framework type of a model.
+type ModelFramework int
+
+const (
+	// ModelFrameworkUnspecified is an unspecified model framework.
+	ModelFrameworkUnspecified ModelFramework = iota
+	// ModelFrameworkTFLite specifies a TFLite model framework.
+	ModelFrameworkTFLite
+	// ModelFrameworkTensorFlow specifies a TensorFlow model framework.
+	ModelFrameworkTensorFlow
+	// ModelFrameworkPyTorch specifies a PyTorch model framework.
+	ModelFrameworkPyTorch
+	// ModelFrameworkONNX specifies a ONNX model framework.
+	ModelFrameworkONNX
+)
+
+// Any contains an arbitrary serialized protocol buffer message along with a URL that describes the type of serialized message.
+type Any struct {
```

Comment:
(minor) can we change the name here? `GenericProtoMessage` or something like that would be a bit more descriptive. `Any` seems a bit mesleading, especially since `any` is a reserved keyword in golang that aliases to `interface{}`.